samples/TodoList/src/TodoList.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+* TodoList.tsx
+* Copyright: Microsoft 2017
+*
+* Display the todo items.
+*/
+
+import { ComponentBase } from 'resub';
+import RX = require('reactxp');
+import TodosStore = require('./TodosStore');
+import TodoStyles = require('./TodoStyles');
+
+interface TodoListState {
+    todos?: string[];
+}
+
+class TodoList extends ComponentBase<{}, TodoListState> {
+    protected _buildState(props: {}, initialBuild: boolean): TodoListState {
+        return {
+            todos: TodosStore.getTodos()
+        }
+    }
+
+    render() {
+        return (
+            <RX.ScrollView style={ TodoStyles.styles.todoListScroll} >
+                { this.state.todos.map(x =>
+                    <RX.View key={x} style={ TodoStyles.styles.todoListItemCell }>
+                        <RX.Text key={x} style={ TodoStyles.styles.todoListItemText }>
```

Comment:
nit: nested RX.Text doens't require a key since it's relative location to it's parent may never change